packages/ripple-address-codec/README.md

```diff
@@ -67,7 +67,7 @@ Check whether a classic address (starting with `r`...) is valid.
 
 Returns `false` for X-addresses (extended addresses). To validate an X-address, use `isValidXAddress`.
 
-### encodeSeed(entropy: Buffer, type: 'ed25519' | 'secp256k1'): string
+### encodeSeed(entropy: UInt8Array, type: 'ed25519' | 'secp256k1'): string
```

Comment:
Make sure to list all these changed functions in the migration guide if you haven't already :)

Reply:
You can, but I think the migration guide is far more useful if the specific function names are available - that way when someone sees "oh shoot, `encodeSeed` is broken, 'how do I migrate encodeSeed?'" they find the migration guide, and specific instructions for it. 

The fix should be outlined in one spot (how to replace `Buffer` with `UInt8Array` in the common functions like hexToBytes, Buffer.alloc(...), etc., but then having a list at the bottom of that section with "affected functions" can help.